parachains/runtimes/collectives/collectives-polkadot/src/ambassador/tracks.rs

```diff
@@ -0,0 +1,162 @@
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! The Ambassador Program referendum voting tracks.
+
+use super::Origin;
+use crate::{Balance, BlockNumber, RuntimeOrigin, DAYS, MINUTES, UNITS};
+use sp_runtime::Perbill;
+
+/// Referendum TrackId type.
+pub type TrackId = u16;
+
+/// Referendum tracks ids.
+pub mod constants {
+	use super::TrackId;
+
+	pub const CANDIDATE: TrackId = 0;
```

Comment:
Curious if it's not better to have `TrackId` as enum, it's going to be more user-friendly (readable) and easier to understand/migrate if needed at some point.

Reply:
This requires at least one fix in substrate, I will prepare the PR.
But this should not block this PR.

Reply:
Seems like this still needs to be resolved, I'm not sure how the PR above touches on TrackId though. Care to elaborate?

Reply:
The implementation supposed to use this type, uses another one by mistake.
If track id is not u16, this just wont compile. I explain it in the PR's description. 
And I think having track id as enum is just nice to have, just a matter of taste.